integration_tests/mockApis/offenderSearch.ts

```diff
@@ -0,0 +1,63 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+export default {
+  stubGetPrisoners: (): SuperAgentRequest => {
+    const results = {
+      totalPages: 1,
+      totalElements: 2,
+      content: [
+        {
+          lastName: 'test',
```

Comment:
Might be worth using distinct first/last names in the dummy data to be sure we're getting the them the right way round when they're outputted as `SURNAME, FIRSTNAME`!

Reply:
Will do when I get onto using them, just ripped them from existing tests.

Reply:
👍